app/Http/Services/PurchaseOrderService.php

```
@@ -27,10 +29,46 @@ public static function createPurchaseOrderFromCanvass(RequestCanvassSummary $req
+                'uom_id'               => $item->uom,
+                'unit_price'           => $item->unit_price,
+                'serve_status'         => ServeStatus::UNSERVED,
+                'remarks'              => $item->remarks,
+                'metadata'             => [
+                    'po_id' => $requestPurchaseOrder->id,
+                    'po_item_id' => $item->id,
+                ],
+            ]);
+            $mrr->items()->createMany($mappedItems->toArray());
+            return $mrr;
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'createMrrFromPurchaseOrder'. (Clean Code Rules)

(StaticAccess)